PopPUNK/visualise.py

```diff
@@ -60,6 +65,10 @@ def get_options():
                              'are from ref-query')
     iGroup.add_argument('--distances',
                         help='Prefix of input pickle of pre-calculated distances')
+    iGroup.add_argument('--sparse-distances',
+                        help='File (with a name *_rank*_fit.npz) storing a sparse'
+                             ' distance matrix',
+                        default=None)
```

Comment:
Don't we just want one or the other (i.e. we always have exactly one of `--distances` or `--sparse-distances`?). In which case I would suggest removing this new argument, and allowing the sparse matrix via `--distances`, checking its name and type on load

Reply:
At the moment `--sparse-distances` is used to calculate an MST, then `--distances` is used to calculate the remaining links to make it a single continuous network. Alternative is to calculate these distances if we don't store all-v-all - will implement once we have more confidence in the final algorithm.